tcl plugin: Boost headers not included

I get
```
libelektra/src/plugins/tcl/tcl.cpp:18:10: fatal error: 'boost/spirit/include/qi_expect.hpp' file not found
#include <boost/spirit/include/qi_expect.hpp>
         ^
```
on OS X 10.11.4 as well as FreeBSD 10.3-RELEASE.

--- a/src/plugins/tcl/CMakeLists.txt
+++ b/src/plugins/tcl/CMakeLists.txt
@@ -1,5 +1,6 @@
 find_package (Boost)
 if (Boost_FOUND)
+	include_directories (${Boost_INCLUDE_DIR})
 	add_plugin(tcl
 		CPP
 		SOURCES

